src/Documentation/grains/transactions.md

```diff
@@ -3,82 +3,110 @@ layout: page
 title: Transactions in Orleans 2.0
 ---
 
-# Orleans Transactions (Beta)
+# Orleans Transactions
 
-__Orleans transactions are in a beta state and should not be used in production systems.__
-
-Orleans supports distributed transactions against persistent grain state.
+Orleans supports distributed ACID transactions against persistent grain state.
 
 ## Setup
 
-Orleans transactions are opt-in.  A silo must be configured to use transactions.  If it is not, any calls to transactional grains will receive an OrleansTransactionsDisabledException.
-
-### Transaction Manager
-
-Orleans transactions requires a transaction manager which is authoritative on the state of a transaction.  Currently, the transaction manager is run within the cluster and must be configured.  This can be programmatically configured on the silo host builder using UseInClusterTransactionManager and TransactionsConfiguration.
-
-The TransactionsConfiguration contains the following configurable values:
-* TransactionIdAllocationBatchSize - To avoid a storage call on every transaction start, transaction Ids are allocated in batches. This is the number of new transaction Ids allocated per transaction Id generation request (storage call).
-* AvailableTransactionIdThreshold - A new batch of transaction Ids will be automatically allocated if the available ids drop below this threshold.
-* TransactionRecordPreservationDuration - How long to preserve a transaction record in the TM memory after the transaction has completed.  This is used to answer queries about the outcome of the transaction.
+Orleans transactions are opt-in.
+A silo must be configured to use transactions.
+If it is not, any calls to transactional methods on grains will receive an `OrleansTransactionsDisabledException`.
+To enable transactions on a silo, call `UseTransactions()` on the silo host builder.
+`var builder = new SiloHostBuilder().UseTransactions();`
 
```

Comment:
The user also needs to configure storage

C# code should be placed inside a block comment with syntax highlighting (triple back-tick)

For DocFx we have to specify C# like this (as far as I know:
    ``` csharp

Reply:
For the storage configuration addition, is mentioning the need to do so enough? Or should there be another subsection about it - @jason-bragg?


I fixed the markdown code formatting.

Reply:
Storage has it's onw section under setup.